### PR TITLE
Visual changes for cleaner appearance.

### PR DESCRIPTION
**Pull request by joeyk710**
A lot of these changes are made so the commands and visual looks seem cleaner when using the bot.

### CODE REVIEW

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
Why did you changed the support

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # These are supported funding model platforms
 
-ko_fi: sudhan
```
Comment (SudhanPlayz) [inline .github/FUNDING.yml:3]: You just removed mine

File: README.md [hunk 2]
```diff
@@ -1,28 +1,44 @@
 <h1 align="center"><img src="./assets/Music.gif" width="30px"> Advanced Discord Music Bot <img src="./assets/Music.gif" width="30px"></h1>
-<p align="center">Thanks for 4k Views! I made this for you all!</p>
-<div align="center"><img src="./assets/banner.gif"></div>
+
+Disclaimer - This is a fork of the repository https://github.com/SudhanPlayz/Discord-MusicBot. Any new changes from there will be applied on top of any changes I make to my version.
```
Comment (SudhanPlayz) [inline README.md:3]: This is a main repo not a fork

File: README.md [hunk 3]
```diff
@@ -1,28 +1,44 @@
...
-<p align="center">Thanks for 4k Views! I made this for you all!</p>
-<div align="center"><img src="./assets/banner.gif"></div>
+
+Disclaimer - This is a fork of the repository https://github.com/SudhanPlayz/Discord-MusicBot. Any new changes from there will be applied on top of any changes I make to my version.
 
 ## 📝 Latest Announcement
 Discord-MusicBot v4 will be launched shortly :tada: You can able to check out the [project](https://github.com/SudhanPlayz/Discord-MusicBot/projects/1) and its [discussion](https://github.com/SudhanPlayz/Discord-MusicBot/discussions/72)
 
 ## 📝 Tutorial
-Tutorial has been uploaded on YouTube <img src="https://www.youtube.com/about/static/svgs/icons/brand-resources/YouTube_icon_full-color.svg?cache=f2ec7a5" width="30px"> Watch it by clicking [here](https://youtu.be/DQXuTaYa31I)
```
Comment (SudhanPlayz) [inline README.md:9]: No need of this

File: README.md [hunk 4]
```diff
@@ -1,28 +1,44 @@
...
+       git clone https://github.com/joeyk710/Discord-MusicBot.git
+       ```
+
+  2) Insert your bot token into the `.env` file. 
+  
+     If you have issues getting a bot token, follow this guide - https://github.com/joeyk710/Discord-MusicBot/wiki/How-to-get-a-Discord-Bot-token
+
+  3) Open an integrated terminal (if using Visual Studio) and type ```npm i```
 
-## 📝 [Support Server](https://discord.gg/a9SHDpD)
```
Comment (SudhanPlayz) [inline README.md:11]: Why did you removed support server, its an important thing

**Comment by joeyk710** [comment]
I'm not using this for people to use. I just changed it for my personal preference. You don't have to choose any changes you don't want

File: README.md [hunk 5]
```diff
@@ -1,28 +1,44 @@
...
+More commands coming in Version 4 from https://github.com/SudhanPlayz/Discord-MusicBot/projects
+![Screenshot (92)](https://user-images.githubusercontent.com/71621973/112220721-f5652c80-8bfc-11eb-8225-42ada02d3656.png)
 
-<div align="left"><img src="https://pika-pika.is-inside.me/zJduq9Gn.png"></div><div align="center"><img src="https://pika-pika.is-inside.me/9ufBoGDH.png"></div><div align="right"><img src="https://pika-pika.is-inside.me/8QkGROJv.png"></div>
 
 ## 💨 Run the projects
-Glitch: [![Remix on Glitch](https://cdn.glitch.com/2703baf2-b643-4da7-ab91-7ee2a2d00b5b%2Fremix-button.svg)](https://glitch.com/edit/#!/import/github/SudhanPlayz/Discord-MusicBot)
-Repl: [![Run on Repl.it](https://repl.it/badge/github/SudhanPlayz/Discord-MusicBot)](https://repl.it/github/SudhanPlayz/Discord-MusicBot)
-Heroku: [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/SudhanPlayz/Discord-MusicBot)
-## ✨ Contributors
```
Comment (SudhanPlayz) [inline README.md:22]: Why did you removed contributers

File: app.json [hunk 6]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "Discord-MusicBot",
-  "description": "Very simple discord music bot with the discord.js with Song Name playing. It can able to play music with the song name",
-  "repository": "https://github.com/SudhanPlayz/Discord-MusicBot",
-  "logo": "https://cdn.discordapp.com/avatars/750613142488481843/e6326038dbe2243ca551ba5b6ecd8bf2.png?size=1024",
+  "description": "Very simple discord music bot with very easy-to-use commands and interface.",
+  "repository": "https://github.com/joeyk710/Discord-MusicBot",
```
Comment (SudhanPlayz) [inline app.json:4]: Why did you changed as your repo

File: commands/help.js [hunk 7]
```diff
@@ -17,10 +17,11 @@ module.exports = {
         })
 
         let embed = new MessageEmbed()
-        .setAuthor("Commands of "+client.user.username, "https://raw.githubusercontent.com/SudhanPlayz/Discord-MusicBot/master/assets/Music.gif")
-        .setColor("BLUE")
-        .setDescription(allcmds)
-        .setFooter(`To get info of each command you can do ${client.config.prefix}help [command] | By ItzCutePikachu#5949`)
```
Comment (SudhanPlayz) [inline commands/help.js:23]: Why did you removed my footer

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
You really just changed everything as your own, I won't accept merging

File: .github/FUNDING.yml (hunk 1 above)
Comment (joeyk710) [reply .github/FUNDING.yml:3]: This file changed on it's own actually

Comment (SudhanPlayz) [reply .github/FUNDING.yml:3]: I not gonna merge

**Pull request closed without merging**